### PR TITLE
Category models and relationship

### PR DESCRIPTION
**Pull request by LevinGeorge**

### CODE REVIEW

File: db/migrate/20160314123046_create_categories.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration
+  def change
+    create_table :categories do |t|
+      t.string :title
+
```
Comment (danielpaul) [inline db/migrate/20160314123046_create_categories.rb:5]: Could we also have a description column here?

**Comment by LevinGeorge** [comment]
@danielpaul I didn't touch the active admin :P

**Comment by danielpaul** [comment]
@LevinGeorge Could you touch the active admin and add the model to active admin too pls? :P

**Comment by LevinGeorge** [comment]
@danielpaul Yeah I'm looking into it now :)

File: app/admin/product.rb [hunk 2]
```diff
@@ -1,3 +1,16 @@
 ActiveAdmin.register Product do
+  permit_params :title, :content
```
Comment (danielpaul) [inline app/admin/product.rb]: Categories and other details about the product should also be editable?

Comment (LevinGeorge) [inline app/admin/product.rb]: Done :+1:

**Comment by LevinGeorge** [comment]
@danielpaul You renamed the PR!? I was going to do that.. Nice try daniel :+1:

**Pull request closed without merging**